dll/win32/syssetup/wizard.c

```diff
@@ -509,6 +629,13 @@ DoWriteProductOption(PRODUCT_OPTION nOption)
         goto Error;
     }
 
+    if (nOption == PRODUCT_OPTION_WORKSTATION)
+    {
+        /* Write system sound events values for Workstation */
+        DoWriteSoundEvents(HKEY_CURRENT_USER, s_szDefaultSoundEvents, (LPCWSTR *)s_DefaultSoundEvents[_countof(s_DefaultSoundEvents)][2], _countof(s_DefaultSoundEvents));
+        DoWriteSoundEvents(HKEY_CURRENT_USER, s_szExplorerSoundEvents, (LPCWSTR *)s_ExplorerSoundEvents[_countof(s_ExplorerSoundEvents)][2], _countof(s_ExplorerSoundEvents));
```

Comment:
Hmm... I don't get the idea of what you are trying to pass as the function 3rd argument, i.e. `(LPCWSTR *)your_array[_countof(your_array)][2]` - what are you trying to achieve by this? :thinking:

Reply:
It's indeed seems to be wrong, sorry. It was just my unsuccessful attempts to fix the compilation. :trollface: